Library/Formula/awscli.rb

```diff
@@ -85,13 +85,16 @@ def install
 
     system "python", *Language::Python.setup_install_args(libexec)
 
+    # Install bash completion
+    bash_completion.install "bin/aws_completer"
+
     # Install zsh completion
     zsh_completion.install "bin/aws_zsh_completer.sh" => "_aws"
 
     # Install the examples
     (share+"awscli").install "awscli/examples"
 
-    bin.install Dir[libexec/"bin/*"]
+    bin.install libexec/"bin/aws"
     bin.env_script_all_files(libexec+"bin", :PYTHONPATH => ENV["PYTHONPATH"])
```

Comment:
Sorry, I had looked for an `env_script_file` or `env_script_single_file`, but didn't think of looking with a different prefix.